### PR TITLE
fix: run install comands for api

### PR DESCRIPTION
**Pull request by aaa006bd**
install depnedency before starting the python server

### CODE REVIEW

File: .gitpod.yml [hunk 1]
```diff
@@ -38,5 +39,5 @@ ports:
   - name: App
     port: 5137
```
Comment (anbraten) [inline .gitpod.yml:40]:
```suggestion
    port: 5173
```

File: .gitpod.yml [hunk 2]
```diff
@@ -21,9 +21,10 @@ tasks:
       nvm use default
     command: |
       pnpm start
```
Comment (anbraten) [inline .gitpod.yml:23]: @aaa006bd Could you add this to the Setup step:

```yml
      env:
          NUXT_TELEMETRY_DISABLED: '0' 
```

Comment (aaa006bd) [reply .gitpod.yml:23]: @anbraten sure

File: .gitpod.yml (hunk 1 above)
Comment (aaa006bd) [reply .gitpod.yml:40]: @anbraten the app runs in port 3000 though?

File: .gitpod.yml [hunk 3]
```diff
@@ -21,9 +21,12 @@ tasks:
...
-  - name: Run backend
+      env:
+          NUXT_TELEMETRY_DISABLED: '0'
```
Comment (anbraten) [inline .gitpod.yml:25]: This needs to be in the setup task area and currently the indention is wrong

Comment (aaa006bd) [reply .gitpod.yml]: @anbraten done here cba95a6583520243450e95ceac44978ec25bf17c

File: .gitpod.yml [hunk 4]
```diff
@@ -2,6 +2,8 @@ image: gitpod/workspace-full-vnc:latest
 
 tasks:
   - name: Setup
+    env:
+          NUXT_TELEMETRY_DISABLED: '0'
```
Comment (anbraten) [inline .gitpod.yml:6]:
```suggestion
        NUXT_TELEMETRY_DISABLED: '0'
```

**Pull request merged**